runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -529,6 +532,34 @@ public void focusLost(FocusEvent e)
 				item.add(button, BorderLayout.EAST);
 			}
 
+			if (cid.getType() == File.class)
+			{
+				JFileChooser fileChooser = new JFileChooser();
+				fileChooser.setDialogTitle("Select a custom cursor image");
```

Comment:
This should not be specific to cursor plugin

Reply:
yeah I realized all this right after submitting. will fix up later today